mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -155,15 +176,16 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
           )
           : (
             dstTokens.map((item) => (
-              <BalanceItemRow
-                key={item.symbol}
-                onPress={() => navigation.navigate({
-                  name: 'TokenDetail',
-                  params: { token: item },
-                  merge: true
-                })}
-                token={item}
-              />
+              <View key={item.symbol} style={tailwind('p-2 pb-1.5')}>
```

Comment:
The spacing is wider on figma, and to align vertically with DFI balance card
Need to readjust the padding value here

![image](https://user-images.githubusercontent.com/40191153/151284017-877ce624-ec19-411b-a598-ada4ff3989e2.png)